### PR TITLE
changed copy in document-field guide

### PR DESCRIPTION
**Pull request by raveling**

### CODE REVIEW

File: docs/pages/guides/document-fields.mdx [hunk 1]
```diff
@@ -64,11 +64,11 @@ query {
 }
 ```
 
-Whenever a value is saved to the document field, the extra data from the selection and the label on relationships are removed if they exist so only the id is stored.
+Whenever a value is saved to the document field, the extra data from the selection and the label on relationships are removed (if they exist) so only the id is stored.
 
 ## Component Blocks
 
-Component blocks allow you to add custom blocks to the editor that can accept unstructured content and render a form that renders arbitrary React components for input.
+Component blocks let you to add custom blocks to the editor that can accept unstructured content and render a form that renders arbitrary React components for input.
```
Comment (timleslie) [inline docs/pages/guides/document-fields.mdx:71]:
```suggestion
Component blocks let you add custom blocks to the editor that can accept unstructured content and render a form that renders arbitrary React components for input.
```

File: docs/pages/guides/_doc-field-intro.mdx [hunk 2]
```diff
@@ -1,5 +1,11 @@
-The [`document`](../apis/fields#document) field type provides a highly configurable rich content field based on [Slate](https://docs.slatejs.org/).
-The Admin UI includes a document editor, which allows content creators to quickly and easily add content to your system.
-You can try out the document editor below.
+The [`document`](../apis/fields#document) field type is a highly customizable rich text editor that lets content creators quickly and easily edit content in your system.
+
+It's built with [Slate](https://docs.slatejs.org/), stores your content as JSON-structured data, and lets you to things like:
```
Comment (JedWatson) [inline docs/pages/guides/_doc-field-intro.mdx:3]:
```suggestion
It's built with [Slate](https://docs.slatejs.org/), stores your content as JSON-structured data, and lets you do things like:
```

**Review by JedWatson — DISMISSED** [review]
- Configure your own custom marks and annotations

... this isn't really something you can do yet. We're going to add it in a future release, but not before the fields refactor is further along.

The rest of it could be expanded on further though, maybe like this:

* Embed multi-column layouts (that your front-end code can decide how to render)
* Insert relationships to other items in your Keystone database
* Define your own custom blocks based on React Components
* Choose between automatically generated forms for editing Component Props, or create your own custom Component UI and Toolbars

... this part about custom blocks is where I think we should really be talking about design systems, but that needs more thought & could come in a later edit

**Review by raveling — COMMENTED** [review]
I think these are all in the most up to date version

**Review by dominikwilkowski — APPROVED** [review]
Looks good

**Pull request merged**